README.md

```diff
@@ -426,9 +411,36 @@ NPM js proxy will cache all artifacts from npmjs, so if somebody decides to remo
 npm --registry=http://central.artipie.com/npmjs/ install npm
 ```
 
-### Credentials settings
+## Authentication
+
+For now, we have two ways to configure authentication in Artipie: via environment variables and via 
+yaml credentials file. Authentication type should be set in main Artipie config inside `meta`
+section:
+```yaml
+credentials:
```

Comment:
@olenagerasimova I'd mention that it should be under `meta` element to avoid confusion:
```yaml
meta:
  credentials:
    ...
```